packages/react-storage/src/components/StorageBrowser/controls/hooks/__tests__/useTitle.spec.tsx

```diff
@@ -0,0 +1,40 @@
+import { useControlsContext } from '../../../controls/context';
+import { useTitle } from '../useTitle';
+
+jest.mock('../../../controls/context');
+
+describe('useTitle', () => {
+  const data = {
+    title: 'ShinyNewTitle',
+  };
+
+  const emptyData = {};
+
+  const actionsConfig = {
+    isCancelable: true,
+    type: 'BATCH_ACTION',
+  };
+
+  // assert mocks
+  const mockUseControlsContext = useControlsContext as jest.Mock;
```

Comment:
Let's try to use `jest.mocked` moving forward so that we maintain some typing information
```suggestion
  const mockUseControlsContext = jest.mocked(useControlsContext);
```